src/BarChart/BarChartHorizontalMultiSeries.story.tsx

```diff
@@ -63,6 +63,37 @@ export const Simple = () => (
   />
 );
 
+export const BarTargetMarker = () => (
+  <BarChart
+    width={500}
+    height={350}
+    data={multiCategory.map((series) => ({
```

Comment:
I would move this out of the function, also I try to avoid dynamic data because its not as easy to compare.